lib/lrama/grammar/parameterizing_rules/builder/separated_nonempty_list.rb

```diff
@@ -2,6 +2,7 @@ module Lrama
   class Grammar
     class ParameterizingRules
       class Builder
+        # Builder for separated list of general parameterizing rules
```

Comment:
"Builder for separated nonempty list ..." ?

Reply:
😱 Sorry, I updated this PR.